src/components/Badge/__tests__/__snapshots__/Badge.test.tsx.snap

```diff
@@ -70,7 +70,7 @@ ShallowWrapper {
           fontSize={8.8}
         >
           <text
-            fill="#010101"
+            fill="#030303"
```

Comment:
snapshot shows the new colors are not exactly as before, but I guess the new ones are correct?

Reply:
yes, I updated all snapshots that should correspond to the new colors. #030303 is patternfly "black"... #010101 is not an official patternfly color.